### PR TITLE
[WIP] On-boarding [Demo (Wed, 18/11/21)]

### PR DESCRIPTION
**Pull request by yeskunall**
Demoing our supervisor this Wednesday.

__DO NOT MERGE THIS PR__. I’ll be cherry-picking the commits that are okay to make into `master`. This is some pure garbage otherwise.

### CODE REVIEW

**Comment by yeskunall** [comment]
@abungay pull these changes and let me know what you think.

File: src/components/Forms/Onboarding/index.js [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+        <Box>
+          <Card visible={true}>
+            <h1>
+              Hello{' '}
+              <span role="img" aria-label="waving">
+                👋🏽
+              </span>
+            </h1>
+            <p>
+              Give your blockhain a name. Giving it a description is
```
Comment (abungay) [inline src/components/Forms/Onboarding/index.js]: 'blockhain'

File: src/components/Forms/Onboarding/index.js [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+            </p>
+            <Label>Name the blockchain</Label>
+            <InfoLabel>
+              <Icon type="exclamation-circle" />
+              <p>This can’t be changed later</p>
+            </InfoLabel>
+            <Input
+              name="name"
+              onChange={this.handleChange}
+              placeholder="Name of blockhain"
```
Comment (abungay) [inline src/components/Forms/Onboarding/index.js]: blockhain

File: src/components/Forms/Onboarding/index.js (hunk 1 above)
Comment (yeskunall) [reply src/components/Forms/Onboarding/index.js]: Lmao o shit. I’ll fix these typos!

**Pull request closed without merging**